draft-ietf-acme-acme.md

```diff
@@ -779,9 +779,11 @@ website (optional, string):
 information about the ACME server.
 
 caaIdentities (optional, array of string):
-: Each string MUST be a lowercase hostname which the ACME server recognizes as
-referring to itself for the purposes of CAA record validation as defined in
-{{!RFC6844}}.  This allows clients to determine the correct issuer domain name to
+: The hostnames that the ACME server recognizes as referring to
+itself for the purposes of CAA record validation as defined in
+{{!RFC6844}}.  Each string MUST represent the same sequence of ASCII
+code points that the server will expect to see in a CAA record.
```

Comment:
I think we should borrow RFC6844's terminology and be a little bit more precise here. Something like:
"Each string MUST represent the same sequence of ASCII code points that the server will expect to see as the "Issuer Domain Name" in a CAA issue or issuewild property tag"

Reply:
👍